pyccel/errors/messages.py

```diff
@@ -163,4 +163,6 @@
 KERNEL_STACK_ARRAY_ARG = "A variable allocated on the stack can't be passed to a Kernel function"
 NON_KERNEL_FUNCTION_CUDA_VAR = 'Cuda internal variables should only be used in Kernel or Device functions'
 UNVALID_KERNEL_CALL_BLOCK_NUM = 'Invalid Block number parameter for Kernel call'
-UNVALID_KERNEL_CALL_TP_BLOCK = 'Invalid Thread per Block parameter for Kernel call'
\ No newline at end of file
+UNVALID_KERNEL_CALL_TP_BLOCK = 'Invalid Thread per Block parameter for Kernel call'
```

Comment:
i think you forgot to use these.

Reply:
Fixed 0d7b325d0a6c14dde370d98ad3934ab812361a6d